.gitignore

```diff
@@ -86,3 +86,6 @@ typings/
 
 # DynamoDB Local files
 .dynamodb/
+
+# GitHub files
+.github
```

Comment:
Why remove .github files?

Reply:
So when I clone, it comes  .github files that I would not want to push to the remote with the same files

Reply:
> Why remove .github files?

So I figured I wasn't supposed to remove them

Reply:
🤝